CONTRIBUTING.adoc

```diff
@@ -0,0 +1,29 @@
+[[contribution-guidelines]]
+= Contribution guidelines
+
+[[filing-issues]]
+== Filing issues
+
+File issues using the standard
+https://github.com/minishift/minishift-addons/issues[Github issue tracker] for the repository.
+Before you submit a new issue, we recommend that you search the list of issues to see if anyone already submitted a similar issue.
+
+[[contributing-patches]]
+== Contributing patches
+
+Thank you for your contributions! Please follow this process to submit a patch:
+
+.  Create an issue describing your proposed change to the repository.
+.  Fork the repository and create a topic branch.
+.  Submit a pull request with the proposed changes.
+
+NOTE: If you submit a new add-on, the add-on must be added as a sub-directory in the root of the repository.
```

Comment:
Or we make it:

"If you submit a new add-on, the add-on must be added as a sub-directory to the _add-ons_ directory."